packages/dx-react-scheduler-material-ui/src/templates/layouts/horizontal-view-layout.jsx

```diff
@@ -30,15 +29,18 @@ export class HorizontalViewLayoutBase extends React.PureComponent {
       classes,
       layoutRef,
       layoutHeaderRef,
+      height,
     } = this.props;
 
+    const containerStyle = height ? { height: `${height}px` } : undefined;
```

Comment:
Can we provide a way to set auto height?

Reply:
What do you mean by `auto`?

Reply:
I mean 100% of a container's height. I know it can be done using AutoSizer but if it's not hard we should try to implement it

Reply:
Stretch to parent element?

Reply:
Okay, we should try